tools/png2h.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
```

Comment:
I'd rename this python script, since it targets png cursors only.
Something like `png2cursorh.py` (I'm bad at naming things)

Reply:
How about `cur2header.py`?

Reply:
or `pngcur2h.py`?

Reply:
That's fine with me :)